docs/packages/margarita-components/index.md

```diff
@@ -3,6 +3,19 @@ title: Margarita Components
 sidebar_label: Margarita Components
 ---
 
-_This page is a stub._
+Margarita Components is a local package to this repository, which contains components that are specific and tailored to the margarita needs and business logic, by composing components from [@kiwicom/universal-components](https://kiwicom.github.io/margarita/docs/packages/universal-components/).
+
+examples of this components would be the `PassengerCard`, or the `CreditCardPaymentForm`. To launch the Storybook in `mobile` version run:
```

Comment:
Start with big `E`. And maybe add extra empty line between first sentence and `To launch...`?